project_event/i18n/fr.po

```diff
@@ -1093,17 +1379,17 @@ msgstr ""
 
 #. module: project_event
 #: model:ir.model.fields,field_description:project_event.field_project_task_responsible_id
-msgid "Responsible"
-msgstr "Responsable de tâche/activité"
+msgid "Task/Activity Responsible"
```

Comment:
Check if we have to keep the old (responsible) or the new one (Task/Activity Responsible)

Reply:
Done